src/Tizen.Multimedia.Remoting/MediaController/MediaControlMetadata.cs

```diff
@@ -116,5 +125,143 @@ internal MediaControlMetadata(IntPtr handle)
+            set => SeasonEncoded = Encode(value, MediaControllerNativeAttribute.Season);
+        }
+
+        /// <summary>
+        /// Gets or sets the episode information.
+        /// </summary>
+        /// <seealso cref="SeriesInformation"/>
+        /// <since_tizen> 6 </since_tizen>
+        public SeriesInformation Episode
+        {
+            get => Decode(MediaControllerNativeAttribute.Episode);
+            set => EpisodeEncoded = Encode(value, MediaControllerNativeAttribute.Episode);
+        }
+
+        /// <summary>
+        /// Gets or sets the content resolution.
+        /// </summary>
+        /// <seealso cref="Size"/>
+        /// <since_tizen> 6 </since_tizen>
+        public Size Resolution
+        {
+            get => Decode();
+            set => ResolutionEncoded = Encode(value.Width, value.Height);
+        }
+
+        // Native CAPI used only encoded string to gets or sets Season, Episode, Resolution.
+        // But encoded string is not useful for user, so we don't offer as it is.
+        // It'll be used internally.
+        internal string SeasonEncoded { get; private set; }
+
+        internal string EpisodeEncoded { get; private set; }
+
+        internal string ResolutionEncoded { get; private set; }
+
+        private string Encode(SeriesInformation information, MediaControllerNativeAttribute type)
+        {
+            string encoded = null;
+
+            if (type == MediaControllerNativeAttribute.Season)
+            {
+                Native.EncodeSeason(information.Number, information.Title, out encoded).
+                    ThrowIfError("Failed to encode season");
+            }
+            else if (type == MediaControllerNativeAttribute.Episode)
+            {
+                Native.EncodeEpisode(information.Number, information.Title, out encoded).
+                    ThrowIfError("Failed to encode episode");
+            }
+
+            return encoded;
+        }
+
+        private string Encode(int width, int height)
+        {
+            Native.EncodeResolution((uint)width, (uint)height, out string encoded).
+                ThrowIfError("Failed to encode resolution");
+
+            return encoded;
+        }
+
+        private SeriesInformation Decode(MediaControllerNativeAttribute type)
+        {
+            int number = 0;
+            string title = null;
+
+            if (SeasonEncoded != null && type == MediaControllerNativeAttribute.Season)
+            {
+                Native.DecodeSeason(SeasonEncoded, out number, out title).
+                    ThrowIfError("Failed to decode season");
+            }
+            else if (EpisodeEncoded != null && type == MediaControllerNativeAttribute.Episode)
+            {
+                Native.DecodeEpisode(EpisodeEncoded, out number, out title).
+                    ThrowIfError("Failed to decode episode");
+            }
+
+            return new SeriesInformation(number, title);
+        }
+
+        private Size Decode()
```

Comment:
Ambiguous api name, usually, `Decode` need a input value
I can't suppose which type was converted to Size type

Reply:
```suggestion
        private Size GetResolution()
```